lib/components/ms-input/MsAddressInput.vue

```diff
@@ -54,8 +55,20 @@ defineExpose({
 const geoapifyApi = new GeoapifyAPI(props.geoapifyApiKey);
 const addressesFound = ref<Array<Address>>([]);
 const address = ref('');
+const lostFocus = ref(true);
 let querying = false;
 
+function handleFocusLost(): void {
+  // Wait a bit to let the click event on the dropdown to be processed
+  setTimeout(() => {
```

Comment:
What's the point of this timeout? Doesn't seem to change anything if I remove it. And code that relies on a 100ms timeout is very probably code that is bugged.

Reply:
If we don't define a timeout, we cannot select an address, so 100ms is fine to make it works.